projects/plugins/crm/includes/jpcrm-templating-placeholders.php

```diff
@@ -1755,6 +1755,16 @@ public function replace_placeholders(
 					
 					}
 
+					// If the key is quote-currency and the value is empty, let's use the currency set in the site settings.
+					if ( $key === 'quote-currency' && $replace_with === '' ) {
+
+						$settings = $zbs->settings->getAll();
+						if ( $settings['currency'] && $settings['currency']['strval'] ) {
+
+							$replace_with = $settings['currency']['strval'];
+						}
+					}
+
```

Comment:
As in the other PR, I don't think we should be putting key-specific fixes here; shouldn't we just pass the proper values to the function?

Reply:
I see what you mean. My original intention was to reduce duplication, hence putting it here. I've moved it back out to the relevant functions that need it: e3199b1c2df3d8674cd3d7f49cb9dc0573528a58